### PR TITLE
Added Nutrition Label comment

### PR DESCRIPTION
**Pull request by jones065**
fix issue #1 - Added a comment for Nutrition Label

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,4 +1,5 @@
 <!DOCTYPE html>
+<!-- Nutrition Label -->
```
Comment (carmen-x) [inline index.html]: I am going to start a code review

**Review by carmen-x — COMMENTED** [review]
Check for comment "I am going to start a code review" to see if you can find it...

File: index.html (hunk 1 above)
Comment (jones065) [reply index.html]: Ok, I see the comments you added!  Can you merge?

**Review by carmen-x — CHANGES_REQUESTED** [review]
Please also add "Typography freecodecamp.org" to the comment

**Review by carmen-x — APPROVED** [review]

**Pull request merged**